django/apps/registry.py

```diff
@@ -151,6 +151,7 @@ def get_app_config(self, app_label):
             for app_config in self.get_app_configs():
                 if app_config.name == app_label:
                     message += " Did you mean '%s'?" % app_config.label
+                    message += " Perhaps a missing dependency on a migration?"
```

Comment:
I'd rather not have a reference to migrations in here. Instead I'd prefer overriding the `StateApps` class in `django.db.migrations.state` to catch the `LookupError` and reraise with an amended message.